packages/token-bridge-sdk/tsconfig.json

```diff
@@ -1,21 +1,9 @@
 {
```

Comment:
It's similar to the ui config no? Can we move all this to the `tsconfig.base.json`? Do we need to have seperate config for now?

Reply:
UI doesn't have `outDir` set and needs to have `noEmit` as `true`
The common ones are all in the base file